src/app/component/ubs/components/ubs-order-details/ubs-order-details.component.html

```diff
@@ -166,7 +166,7 @@ <h5>Чи чекаєте Ви замовлення з нашого Еко-маг
         <div formArrayName="additionalOrders" *ngFor="let order of additionalOrders.controls; let i=index"
           class="form-group">
           <label class="label m-0">Введіть номер замовлення</label>
-          <input [formControlName]="i" class="shadow-none form-control border-input p-2" type="text" placeholder="xxxxxxxxxx">
+          <input [formControlName]="i" class="shadow-none form-control border-input p-2" type="text" placeholder="xxxxxxxxxx" [readonly]="shop.value === 'no'">
```

Comment:
what is 'no' here ? if it some kind of response, please create enum for it

Reply:
'no' is a value of radio input. Do we need an enum for it?

Reply:
i think  we need to move to ts file
someCollName: object = { no: 'no', yes: 'yes' }